client/contracts/bindings/addresses.go

```diff
@@ -0,0 +1,8 @@
+package bindings
+
+var ProxyFactoryAddr = "0x1582aEd4A8156325e28ef9eF075Da1E1D44AA56E"
+var ProxyKNSRegistryAddr = "0xDfA58b86D285cF07E15ceEc0BaAb89e1D26305d7"
+var ProxyRegistrarAddr = "0x927C9AF9CF36a9d43d3160576e750612bc9e63e9"
+var ProxyResolverAddr = "0x577baB773e23327B0Ab7081E2a1ffcd186514d75"
+
+var MultiSigWalletAddr = "0xfE9bed356E7bC4f7a8fC48CC19C958f4e640AC62"
```

Comment:
This one should not be necessary?

Reply:
It is since we need to access to it several times when deploying the other contracts, I tried to remove without success.